src/partition.c

```diff
@@ -1349,13 +1359,348 @@ static void pick_metis(int nodeID, struct space *s, int nregions,
 }
 #endif
 
-#if defined(WITH_MPI) && (defined(HAVE_METIS) || defined(HAVE_PARMETIS))
+#if defined(WITH_MPI) && defined(HAVE_SCOTCH)
+/**
+ * @brief Fill the adjncy array defining the graph of cells in a space.
+ *
+ * See the ParMETIS and METIS manuals if you want to understand this
```

Comment:
I remember that this is all modelled on METIS for compatibility, but is it true that SCOTCH points you at the METIS
documentation?

Reply:
BTW, is it SCOTCH, Scotch or scotch? Always nice to be consistent with the proper name.